sdk/communication/communication-chat/README.md

```diff
@@ -92,7 +92,7 @@ Use `sendMessage` method to sends a message to a thread identified by threadId.
 - Use `senderDisplayName` to specify the display name of the sender;
 - Use `type` to specify the message type, such as 'text' or 'html' ;
 
-`sendChatMessageResponse` is the response returned from sending a message, it contains an ID, which is the unique ID of the message.
+`SendChatMessageResult` is the response returned from sending a message, it contains an ID, which is the unique ID of the message.
```

Comment:
sendChatMessageResult  (lower case the first one - that is the name on the example)

Reply:
Isn't the model name upper-case?